src/common/xo-line-chart/index.js

```diff
@@ -111,7 +119,7 @@ const templateError =
     No stats.
   </div>
 
-// ===================================================================
+// =============================================================================
```

Comment:
I filled it to 80 characters but didn't realise it wasn't consistent with the other dividers. I'm changing it back.